sdk/metric/pipeline_test.go

```diff
@@ -518,3 +536,91 @@ func TestExemplars(t *testing.T) {
 		check(t, r, 2, 2, 2)
 	})
 }
+
+func TestAddingAndObservingMeasureConcurrency(t *testing.T) {
```

Comment:
per https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#testing

```suggestion
func TestAddingAndObservingMeasureConcurrentSafe(t *testing.T) {
```

Reply:
Done